lib/apiservers/engine/backends/container_test.go

```diff
@@ -246,7 +260,7 @@ func (m *MockContainerProxy) AddContainerToScope(handle string, config types.Con
 	return m.mockAddToScopeData[respIdx].retHandle, m.mockAddToScopeData[respIdx].retErr
 }
 
-func (m *MockContainerProxy) AddVolumesToContainer(handle string, config types.ContainerCreateConfig) (string, error) {
+func (m *MockContainerProxy) AddVolumesToContainer(ctx context.Context, handle string, config types.ContainerCreateConfig) (string, error) {
```

Comment:
why context, why not trace.Operation?

Reply:
This one is a toss up.  If we use trace.Operation, external clients will also have to import that.  I can be convinced to use trace.Operation instead.  Let me reconsider this.

Reply:
I'm going to delay the operation tracing changes till a future refactor pass.  Our code uses a lot of context and cancellable context.  I'm afraid the change to op trace would introduce logic changes I don't want to take on in the first pass.  Let's get this PR in and then we can do the op trace in a future pass.